src/test/java/lotto/LottoNumbersTest.java

```diff
@@ -0,0 +1,62 @@
+package lotto;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoNumbers;
+
+class LottoNumbersTest {
+
+    @Test
+    @DisplayName("LottoNumbers 파라미터로 Sting이 들어왔을 때, 값을 잘 분리하는지 확인한다.")
+    void test1() {
+        LottoNumbers lottoNumbers = new LottoNumbers("1,2,3,4,5,6");
```

Comment:
```suggestion
        LottoNumbers lottoNumbers = new LottoNumbers(1, 2, 3, 4, 5, 6);
```
문자열도 지원하지만 위와 같이 숫자도 지원하면 어떨까?
즉, 가변인자를 가지는 생성자를 추가하면 어떨까?